docs/maintainer-guide/working-groups.md

```diff
@@ -0,0 +1,22 @@
+# Working Groups
+
+The ESLint TSC may form working groups to focus on a specific area of the project.
+
+## Creating a Working Group
+
+Working groups are created by a standard TSC motion. Each working group:
+
+1. Must have a GitHub team under the "Working Groups" top-level GitHub team.
+1. Must have at least one TSC member.
+1. May have any number of Committers and Reviewers.
+1. Must have at least two members.
+
+Active working groups are listed on the [team page](https://eslint.org).
```

Comment:
Is this link accurate? (Maybe it's just a placeholder)

Reply:
Oops, I meant for that to be `/team`.